scripts/src/main/resources/scripts/environment-project

```diff
@@ -37,9 +43,16 @@ export JAVA_HOME=$SOFTWARE_PATH/java
 ###
 export M2_HOME=$SOFTWARE_PATH/maven
 export M2_CONF=$OASP_PROJECT_HOME/$CONF_PATH/.m2/settings.xml
-export MAVEN_OPTS="$MAVEN_OPTS -Xmx512m -Duser.home=$OASP_PROJECT_HOME/$CONF_PATH"
+export MAVEN_OPTS="-Xmx512m -Duser.home=$OASP_PROJECT_HOME/$CONF_PATH"
```

Comment:
This ($MAVEN_OPTS) was added by me as a requested feature. Please do not remove it.